Source/carthage/CopyFrameworks.swift

```diff
@@ -31,19 +31,38 @@ public struct CopyFrameworksCommand: CommandType {
 						|> flatMap(.Merge) { (source, target, validArchitectures) -> SignalProducer<(), CarthageError> in
 							return combineLatest(copyProduct(source, target), codeSigningIdentity())
 								|> flatMap(.Merge) { (url, codesigningIdentity) -> SignalProducer<(), CarthageError> in
-									return stripFramework(target, keepingArchitectures: validArchitectures, codesigningIdentity: codesigningIdentity)
+									let strip = stripFramework(url, keepingArchitectures: validArchitectures, codesigningIdentity: codesigningIdentity)
+									if buildActionIsArchiveOrInstall() {
+										return strip
+											|> then(copyBCSymbolMapsForFramework(url, fromDirectory: source.URLByDeletingLastPathComponent!))
+											|> then(.empty)
+									} else {
+										return strip
+									}
 								}
 						}
 				}
 				|> promoteErrors
 				|> waitOnCommand
-
 		case .Usage:
-			return .success(())
+			return .success()
```

Comment:
We are passing empty tuple (`Void`) explicitly in the codebase, could you revert this change?